### DISCUSSION TITLE
Bonus Level World 1 - Icy Island

### DISCUSSION THREAD

**Discussion by sueco2003** [SuperTux/supertux#2314] [Q&A] [answered]
Hello, how do you get to the bonus level on the first world of history mode?? It´s impossible to find the path...
Hope to get some feedback, thank you.

**Comment by Semphriss** [comment, marked as answer]
One level has a secret exit. You need to find that secret exit and finish the level that way. Then, the path to the secret level will reveal itself.

Let me know if you want spoilers :^)

**Reply by sueco2003** [reply]
Can you just please tell me which is the level? the name ahaha :)

**Reply by provigz** [reply]
> Can you just please tell me which is the level? the name ahaha :)

<details>
  <summary>[Spoiler] Level name</summary>
  <br>
   The name of the level is "Icy Valley".
</details>

**Reply by Semphriss** [reply]
@Vankata453 You can use spoilers on GitHub like this:

```html
<details> 
  <summary>Spoiler title</summary>
   Spoiler contents here
</details>
```

<details> 
 <summary>Spoiler title</summary>
  Spoiler contents here
</details>
